required/latex-lab/latex-lab-sec.dtx

```diff
@@ -31,7 +31,7 @@
 % \title{The \textsf{latex-lab-sec} package\\
 % Changes related to the tagging of sectioning commands}
 % \author{\LaTeX{} Project\thanks{Initial implementation done by Ulrike Fischer}}
-% \date{v0.83 2023-06-07}
+% \date{v0.84 2023-06-21}
 %
```

Comment:
is this file missing a \Provides... that the version is in the doc only?

Reply:
I will check